client/lib/abtest/active-tests.js

```diff
@@ -175,5 +175,6 @@ export default {
 		},
 		defaultVariation: 'control',
 		allowExistingUsers: true,
+		localeTargets: 'any',
```

Comment:
Maybe we can use `localeExceptions` here to exclude the `en` locale?
[Link to Doc](https://github.com/Automattic/wp-calypso/blob/add/abtest-for-reskinning-signup-flow/client/lib/abtest/README.md)

Reply:
I learned it today! Thanks @aneeshd16 !